tgui/packages/tgui/interfaces/GasCanister/DetonatorTimer.js

```diff
@@ -0,0 +1,35 @@
+import { Box, AnimatedNumber } from '../../components';
+import { formatTime } from '../../format';
+
+export const DetonatorTimer = props => {
+  const {
+    time,
+    warningThreshold = 30,
+    dangerThreshold = 10,
+    explosionMessage = "BO:OM",
+  } = props;
+
+  const timeColor = () => {
+    if (time <= dangerThreshold) {
+      return "red";
+    } else if (time <= warningThreshold) {
+      return "orange";
+    } else {
+      return "green";
+    }
+  };
+
+  return (
+    <Box
+      p={1}
+      textAlign="center"
+      backgroundColor="black"
+      color={timeColor()}
+      maxWidth="100px"
+      fontSize="19px">
```

Comment:
Can we get away with 18 or 20px? We don't have anything resembling a standard for font-sizes, but pre-emptively trying to bucket them a little better will make this easier down the road.

Reply:
done